src/runtime/threading_backend.cc

```diff
@@ -124,6 +147,62 @@ int MaxConcurrency() {
   return std::max(max_concurrency, 1);
 }
 
+
+unsigned int configThreadGroup(int mode, int nthreads, std::vector<unsigned int> *sorted_order) {
```

Comment:
Use CamelCaseFUnctionName, this can be member of ThreadGroup and combined with SetAffinity